lib/packer/instance.rb

```diff
@@ -38,5 +38,17 @@ def dev_server
     def commands
       @commands ||= Packer::Commands.new self
     end
+
+    private
+
+    def app_root_path
+      if Packer.rails?
+        Rails.root
+      elsif Packer.sinatra?
+        Sinatra::Base.settings.root || Pathname('.')
```

Comment:
Why the fallback to `Pathname('.')`? Is `settings.root` sometimes `nil`?

Reply:
Yes, it might not always be set as part of the Sinatra config. e.g. the `example_app`. Maybe we can be strict and enforce configuration for this to be set?

Reply:
@DXTimer sounds good to me. And then update the example app to work.

Reply:
Turns out that `Sinatra::Base.settings.root` will always be `nil`. However, we could use `Sinatra::Application.settings.root` which should always be a valid path. However in the case that it does return `nil` we should blow up.

Note: This might not be a good approach if the Sinatra application is modular. (Needs to be tested)

Reply:
Seedy is a modular Sinatra application :)